MonoDevelop.MSBuild/Schema/FrameworkInfoProvider.cs

```diff
@@ -51,7 +51,8 @@ public FrameworkInfoProvider ()
 					new VersionInfo (new Version (2, 2), "netcoreapp2.2"),
 					new VersionInfo (new Version (3, 0), "netcoreapp3.0"),
 					new VersionInfo (new Version (3, 1), "netcoreapp3.1"),
-					new VersionInfo (new Version (5, 0), "netcoreapp5.0")
+					new VersionInfo (new Version (5, 0), "netcoreapp5.0"),
+					new VersionInfo (new Version (5, 0), "net5.0")
```

Comment:
Should this include the whole list of TFMs?
https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md#design

Reply:
As a matter of fact, yes!

Reply:
Keep in mind the new TFMs lead down a rabbit hole with all variants and permutations (https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md#os-versions).
I can't find the definition for `VersionInfo`, but ideally you may want to end up with something like `new VersionInfo (new Version (5, 0), "net5.0-*")`

Reply:
I added a regex to account for all of them all at once. Might want to double-check that it will match every valid permutation, though.

Reply:
We'll need to revive this and update for net6.0, net7.0 etc. Prob. need to add the digits to the regex instead of hardcoding `5`.